src/use_picker.js

```diff
@@ -51,16 +48,20 @@ const usePicker = ({
   const selectedBaseItem = items.find((i) => i[valueField] === value);
 
   const selectedItems = _.isArray(value)
-    ? pickerItems.filter((i) => value.includes(i.value))
+    ? items.filter((i) => value.includes(i.value))
```

Comment:
Dit lijkt me niet goed, `items` is namelijk niet genormaliseerd en heeft dus niet altijd een value (da's precies wat pickerItems doet), daarnaast kan items ook een simple array zijn als ['a', 'b', 'c']